src/hex/algorithms.rs

```diff
@@ -1,18 +1,63 @@
+//! Algorithms for an axial coordinate.
```

Comment:
[nitpick] this comment might indicate that the file could have a name that specifies it refers to axial coordinates